src/tfr_control/config/controllers.yaml

```diff
@@ -1,51 +1,60 @@
 # ------------------------------------------------------------
 # This file holds the controller configurations for all controllers
 # on the robot.
+#
+# These are loaded into the controller layer by the  controller
+# manager at will, and the joints they describe need to match the hardware
+# layer
+#
+# The controller manager handles the upper level logic, and makes sure
+# the controllers do not conflict
 # ------------------------------------------------------------
 
 # Publish all joint states -----------------------------------
 joint_state_controller:
-  type: joint_state_controller/JointStateController
-  publish_rate: 50
+    type: joint_state_controller/JointStateController
+publish_rate: 50
```

Comment:
Indentation check, this got unindented: is this correct, does that change any functionality?